src/shared/ui/components/TimeStamp.vue

```diff
@@ -0,0 +1,49 @@
+<script setup lang="ts">
+import { defaultFormat } from '@/shared/lib/time';
+import ContextTooltip from '@/shared/ui/components/ContextTooltip.vue';
+import { useTimeAgo } from '@/shared/ui/composables/useTimeAgo';
+import { computed } from 'vue';
+import { useI18n } from 'vue-i18n';
+
+const props = defineProps<{
+  value: Date
+  inverted?: boolean
+}>();
+
+const { t } = useI18n();
+const res = useTimeAgo(props.value);
```

Comment:
Don't lose reactivity.

```suggestion
const res = useTimeAgo(toRef(props, 'value'));
```